lib/src/manifest.ts

```diff
@@ -72,6 +73,10 @@ export class ManifestBase {
     static LatestVersion = '0.2';
 
     constructor(manifest: XPOCManifest) {
+        const validation = ManifestBase.validate(manifest);
```

Comment:
I'm not sure we should validate manifest in the constructor. Our tools will like parse invalid manifests and we make best effort to construct it. Our validation tools will use the validation functionality to either fix or report manifest issues.